ui/src/virtualconsole/vcxypadfixture.cpp

```diff
@@ -557,7 +557,9 @@ void VCXYPadFixture::readDMX(QList<Universe*> universes, qreal & xmul, qreal & y
         return;
 
     if (m_xReverse == true)
```

Comment:
Thanks for notice me. I saw them too late. These two lines were there only for private test and I forgot them there -.- New pull request will have them removed.